contracts/cyberpunk/src/msg.rs

```diff
@@ -38,4 +38,8 @@ pub enum QueryMsg {
     /// Returns the env for testing
     #[returns(cosmwasm_std::Env)]
     MirrorEnv {},
+
+    /// Queries `DenomMetadata` and `AllDenomMetadata` from the bank module and returns the symbols
+    #[returns((String, Vec<String>))]
+    Denoms {},
```

Comment:
Let's split this in two different cases. That's easier for testing. E.g. you know there is ucosm on the cain, you can query just that.